testsuite/src/main/java/io/netty/testsuite/transport/socket/AbstractClientSocketTest.java

```diff
@@ -34,7 +34,7 @@ protected AbstractClientSocketTest() {
 
     @Override
     protected List<TestsuitePermutation.BootstrapFactory<Bootstrap>> newFactories() {
-        return SocketTestPermutation.INSTANCE.clientSocket();
+        return SocketTestPermutation.INSTANCE.clientSocketWithFastOpen();
```

Comment:
why we change this ? I was under the impression that fast open is only supported by the native epoll transport which means we should override the methods there and specify the right thing .

Reply:
Too eager search and replace.